### PR TITLE
Draft validation checks annex for Ed. 2.3.0

### PR DESCRIPTION
**Pull request by rmalyankar**
Includes draft annex, spreadsheet of "product-specific" validation checks,  and correction to minor typo in clause 10.

### CODE REVIEW

**Review by RohdeBSH — CHANGES_REQUESTED** [review]
I wrote some comments to the file "sources/2.3.0/sections/aj-validation-checks.adoc" which should be answered before the merge.

File: sources/2.3.0/sections/aj-validation-checks.adoc [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+. Checking the presence and correctness of the Exchange Catalogue (CATALOG.XML).
+
+. Verification of signatures, including the Exchange Catalogue signature and signatures for individual datasets, catalogues, and support files.
+
+. Checking the structure and contents of the Exchange Set package, including whether there is a discovery block for each file in the Exchange Set.
+
+. Checking that the metadata encoded in a discovery block and the headers and embedded metadata in the corresponding dataset or Catalogue are compatible.
+
+Generalised checks for Exchange Set validation are being developed by the S-100WG and will be introduced into S-104 when ready.
```
[the remark this answers is not in the document]
Comment (rmalyankar) [reply sources/2.3.0/sections/aj-validation-checks.adoc]: Updated to S-102

File: sources/2.3.0/sections/aj-validation-checks.adoc [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+[[tab-phases-in-validation-processing-for-HDF5-datasets]]
+.Phases in validation processing for HDF5 datasets
+[cols="a,a,a",options="header"]
+|===
+| Phase	| Name	| Description
+
+| 1	| Validate Dataset Root and Feature Information	| Validation of root group of HDF5 file and feature type information.
+
+| 2	| Validate Feature Container Groups	| Validation of metadata and structure for each feature type (“Feature Container”). In S-104 there is only one feature container, so this set of checks is executed only once. If future Editions introduce multiple feature container HDF5 groups, this set must be executed for each feature container HDF5 group.
```
[the remark this answers is not in the document]
Comment (rmalyankar) [reply sources/2.3.0/sections/aj-validation-checks.adoc]: Updated to S-102

File: sources/2.3.0/sections/aj-validation-checks.adoc [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+== Validation
+
+=== Introduction
+The following checks are intended for production systems designed to produce S-102 datasets. The checks can be administered at any time during the production phase. They can also be applied downstream in the distribution and end user systems to test the conformance of a dataset to the format rules specified in S-100 Part 10c and the S-102 Product Specification.
+
+=== Check Classification
+Checks are classified as critical, error, or warning checks as described in the Table below.
+
+[[tab-check-classification]]
+.Classification of checks
```
[the remark this answers is not in the document]
Comment (rmalyankar) [reply sources/2.3.0/sections/aj-validation-checks.adoc:13]: Where in S-100 is this documented? There is no overarching document for validation checks for all product specifications in S-100.

**Comment by rmalyankar** [comment]
Made changes and replied to comments. Do the changes appear in your view of the pull request?

File: sources/2.3.0/sections/aj-validation-checks.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (RohdeBSH) [reply sources/2.3.0/sections/aj-validation-checks.adoc:13]: Then it's okay.
I just thought that this topic is described somewhere in the S-100 documents centrally for all product specifications.

**Review by RohdeBSH — APPROVED** [review]
It's okay.

**Review by hasel001 — APPROVED** [review]

**Pull request merged**